src/zicond.adoc

```diff
@@ -0,0 +1,228 @@
+[[Zicond]]
+== "Zicond" Integer Conditional operations extension
+
+[[intro]]
+=== Introduction
+The Zicond extension defines a simple solution that provides most of the benefit and all of the flexibility one would desire to support conditional arithmetic and conditional-select/move operations, while remaining true to the RISC-V design philosophy.
+The instructions follow the format for R-type instructions with 3 operands (i.e., 2 source operands and 1 destination operand).
+Using these instructions, branchless sequences can be implemented (typically in two-instruction sequences) without the need for instruction fusion, special provisions during the decoding of architectural instructions, or other microarchitectural provisions.
+
+// === Suitability for Fast Track Extension Process
+// This proposed extension meets the Fast Track criteria: it consists of two, simple R-form instructions, it addresses a wide range of use-cases f// or branchless sequences, it composes with the existing RISC-V instruction set, and is not expected to be contentious.
```

Comment:
We can remove the commented-out 'Suitability for Fast Track', given that this has successfully completed the process.  This is a left-over from the original 'fast-track proposal' stage of the fast-track process.

Reply:
Will do.